src/index.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter Challenge</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+    
+
+    <!-- Container Starts-->
+    <div class="container">
+
+        <!-- Header -->
+        <header class="header">
+            <div class="header-logo">
+                <img class="header-logo-img"src="../assets/twitter-color-svgrepo-com.svg" alt="logo">
```

Comment:
Try to create inside assests folder svg folder and contain there just svg it will be better for you to find all of the pictures you need in cor;rect format :)